src/Symfony/Component/Form/DataTransformerInterface.php

```diff
@@ -54,8 +57,10 @@ interface DataTransformerInterface
      * process that value.
      *
      * @param mixed $value The value in the original representation
+     * @psalm-param T|null $value
      *
      * @return mixed
```

Comment:
fabbot is complaining about the `@return mixed`, should I remove it ?

Reply:
No, you can ignore these.